frontend/src/components/edit-profile/select-years-range/select-years-range.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Control } from 'react-hook-form';
+
+import { SelectField } from '@components/common/select-field/select-field';
+import { MenuItem } from '@mui/material';
+
+import styles from './styles.module.scss';
+
+interface RangeYearsProps {
+  start: number;
+  end: number;
+  required: boolean;
+  id?: string;
+  name: string;
+  control: Control;
+  label: string;
+  defaultValue: string;
+}
+
+export const SelectYearsRange: React.FC<RangeYearsProps> = ({
```

Comment:
oh-oh-oh
The naming is totally misleading. Because as result we select the birthday **_year_**. A **_single_** unit of data, not the _**range**_. Also, as the component is named "...Years..." the years' range limitation can be an internal part of the component instead of using the props to pass the range